app/views/layouts/application.html.erb

```diff
@@ -2,8 +2,8 @@
 <html>
```

Comment:
You should add them now so that you can test your current angular views and controllers. Otherwise, you wouldn't be able to test if the current codes are working. :+1: